ddl/install.sql

```diff
@@ -1,4 +1,5 @@
-CREATE OR REPLACE LIBRARY ODBCLoaderLib AS '/opt/vertica/packages/odbc-loader/lib/ODBCLoader.so';
+\set libfile '\''`pwd`'/lib/ODBCLoader.so\'';
```

Comment:
This change would need to be reverted if we revert the lib location change in Makefile.